### PR TITLE
Docs: Deprecation notes 3.4

### PR DESCRIPTION
**Pull request by Simran-B**
Internal Ref:
- https://github.com/arangodb/planning/issues/2932
---

- More prominent deprecation notes for AQL Geo functions

Needs `@HINT` from https://github.com/arangodb/arangodb/pull/5940

### CODE REVIEW

**Comment by Simran-B** [comment]
I got it to produce the following using the new jenkins and https://github.com/arangodb/oskar/pull/12:

image

Only the yellow hint boxes should show in the generated HTML, but in Swagger UI the plain text should show. I'm not sure how/where to do this. @dothebart @ObiWahn

**Comment by Simran-B** [comment]
Tested on c21 using the `buildDocumentation` script (HTTP book only) and updated `generateMdFiles.py` from https://github.com/arangodb/oskar/pull/12 as well as with old `build.sh` in an Ubuntu VM. Both builds were successful.

Could this be done without the field `x-hints` in `api-docs.json`? It's hard to wrap your head around the `generateSwagger.py`, not really sure how to do it without to be honest.

File: Documentation/DocuBlocks/Rest/Collections/delete_api_collection.md [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 @RESTHEADER{DELETE /_api/collection/{collection-name}, Drops a collection}
 
+@HINTS
+{% hint 'warning' %}
```
Comment (dothebart) [inline Documentation/DocuBlocks/Rest/Collections/delete_api_collection.md]: shouldn't these then be the comments? or do you change that on the way into the swagger json?

Comment (Simran-B) [reply Documentation/DocuBlocks/Rest/Collections/delete_api_collection.md]: That is not really possible. The hint type is taken from the Gitbook hint tag, and there can be multiple hint boxes in the body of `@HINTS`. The hint type is prepended to each hint body:

```
@HINTS
There can be plain text here.

{% hint 'danger' %}
Serious warning!
{% endhint %}

{% hint 'info' %}
Something useful.
{% endhint %}
```

becomes:

There can be plain text here.
**Danger:** Serious warning!
**Info:** Something useful.

**Comment by Simran-B** [comment]
Resolved conflict, re-tested and working. Also tested https://github.com/arangodb/oskar/pull/12 again without issues. Please merge both PRs.

**Comment by Simran-B** [comment]
@sleto-it Did you add the WIP label to indicate that the oskar PR needs to be merged first?

**Comment by sleto-it** [comment]
@Simran-B the .0 we discussed last week.. Thanks,

**Comment by Simran-B** [comment]
@sleto-it Right, changed it to 3.4.0 and 4.0.0 (although you may not see this commit on GitHub at the moment). Any idea if merging via GitHub is save if it doesn't show all commits?

**Comment by sleto-it** [comment]
@Simran-B before merging, I will wait green status from https://status.github.com/messages

**Pull request merged**